pom.xml

```diff
@@ -564,10 +563,26 @@
                         <dependency>
                           <groupId>com.puppycrawl.tools</groupId>
                           <artifactId>checkstyle</artifactId>
-                          <version>6.19</version>
+                          <version>${checkstyle.version}</version>
```

Comment:
version should be defined once and for all under pluginManagement so it doesn't have to be specified in plugin at all.

Reply:
follow up in https://github.com/confluentinc/common/pull/92